vulyk/control.py

```diff
@@ -227,4 +243,5 @@ def batch(batch_name, task_type):
         pt.add_row(values)
 
     print(pt)
+
```

Comment:
Why did you add two spaces above and just a single one - here?
FYI: I use those because JetBrains' IDE family do know how to fold these `region/endregion` lovely to make them look pretty.